File: src/main/org/nlogo/prim/etc/_showlink.java
package org.nlogo.prim.etc;

import org.nlogo.agent.Link;
import org.nlogo.nvm.Command;
import org.nlogo.api.Syntax;

public final strictfp class _showlink
    extends Command {
  @Override
  public Syntax syntax() {
    return Syntax.commandSyntax("---L", true);
  }

  @Override
  public void perform(final org.nlogo.nvm.Context context) {
    ((Link) context.agent).hidden(false);
    context.ip = next;
  }
}

File: netlogo-gui/src/main/prim/etc/_showlink.scala
// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo

package org.nlogo.prim.etc

import org.nlogo.agent.Link
import org.nlogo.nvm.{ Command, Context }

class _showlink extends Command {


  switches = true
  override def perform(context: Context): Unit = {
    context.agent.asInstanceOf[Link].hidden(false)
    context.ip = next
  }
}
